src/riscv.h

```diff
@@ -168,7 +168,19 @@ void rv_debug(riscv_t *rv);
 /* step the RISC-V emulator */
 void rv_step(riscv_t *rv, int32_t cycles);
 
-/* get RISC-V user data bound to an emulator */
+/*
+ * @rv: RICV-V
```

Comment:
In this context, `rv` represents the instance of the RISC-V emulator.

Reply:
Since we drop the `rv_userdata`, no more comment about them.